charts/ia-case-api/values.preview.template.yaml

```diff
@@ -9,11 +9,11 @@ java:
     POSTGRES_PASSWORD: hmcts
     POSTGRES_CONNECTION_OPTIONS: ""
     #Un comment APIs if you need to test a local version
-    #IA_CASE_DOCUMENTS_API_URL: https://${SERVICE_NAME}-documents-api.preview.platform.hmcts.net
+    IA_CASE_DOCUMENTS_API_URL: https://${SERVICE_NAME}-documents-api.preview.platform.hmcts.net
```

Comment:
According to the comment, I presume this needs to be reverted.

Reply:
Before merge into master, it will be reverted